dashboards/grafana-dashboard-rhtap-slos.configmap.yaml

```diff
@@ -415,7 +545,7 @@ data:
         ]
       },
       "time": {
-        "from": "now-28d",
+        "from": "now-1h",
```

Comment:
The default evaluation window for SLOs is 28d so please don't change this.

Reply:
Fixed, not sure what happened there, must've forgotten to change back to 28d